File: amicable.py
def amicable_numbers(n1,n2):
    if find_div(n1) == n2 and find_div(n2) == n1:
        return True
    else:
        return False
    
    
def find_div(num):
    arr = []
    i = 1
    while i < num:
        if num % i == 0:
            arr.append(i)
        i = i + 1
    total = sum(arr)
    return total

File: amicable.js
function amicableNumbers(num1, num2) {
    if ((findDiv(num1) == num2 ) && (findDiv(num2) == num1)){
      return true;
    }
    return false;
    
  }
  
  function findDiv (num){
    let arr = [];
    for (let i = 1; i < num; i++){
      if (num % i == 0){
        arr.push(i);
      }
    }
    let sum = arr.reduce((a,b) => a + b, 0);
    return sum;
  }